### PR TITLE
Document feature that allows app-specific key overrides of a generic key map

### PR DESCRIPTION
**Pull request by berwynhoyt**
This documents an accidental feature that allows keys from a generic key map to be overridden for specific applications. Addresses issue #163.

### CODE REVIEW

**Comment by berwynhoyt** [comment]
Would you also like me to edit out the redundancy in [example/config.yml](https://github.com/k0kubun/xremap/blob/master/example/config.yml) that this feature allows?

File: README.md
Comment (k0kubun) [inline README.md:325]:
```suggestion
```

**Comment by k0kubun** [comment]
> Would you also like me to edit out the redundancy in [example/config.yml](https://github.com/k0kubun/xremap/blob/master/example/config.yml) that this feature allows?

Sure, that sounds nice.

**Comment by berwynhoyt** [comment]
Ready to merge, pending your approval.

**Review by k0kubun — APPROVED** [review]

**Pull request merged**
